SAMtags.tex

```diff
@@ -317,11 +320,29 @@ \subsection{Barcodes}
 \subsection{Original data}
 
 \begin{description}
+\item[OA:Z:\tagregex{{\tt (}\sf{RNAME}{\tt,\,}\sf{POS}{\tt,\,}\emph{strand}{\tt,\,}\sf{CIGAR}{\tt,\,}\sf{MAPQ}{\tt,\,}\sf{NM}{\tt; )}+}]
+The tag encodes the original alignment information of the record prior to realignment or unalignment by subsequent tool. 
+Here {\sf strand} encodes the 0x10 bit of {\sf FLAG}---{\sf SEQ} being reverse complemented---such that it is `+' if the bit is unset and `-' if the bit is set,
```

Comment:
I don't know about the relative merits of \`\\{tt -}\' vs \`-' vs \`--' in TeX, but we need to be consistent.  With have both single and double minus used in this document for OA vs SA tag (with the \\tt being the old method).

Reply:
The existing document (notably the barcodes section) uses \`{\tt +}'  \`{\tt -}'  etc. The (TeX!) quotes and \\tt emphasise that they're verbatim characters; and the \\tt makes `+` and `-` the same size. So the relative merits are let's stick with the existing convention.